x-pack/test/functional/apps/transform/creation_runtime_mappings.ts

```diff
@@ -0,0 +1,373 @@
+              'rt_responsetime_x_2.max': {
+                max: {
+                  field: 'rt_responsetime_x_2',
+                },
+              },
+              'rt_responsetime_x_2.min': {
+                min: {
+                  field: 'rt_responsetime_x_2',
+                },
+              },
+              'rt_responsetime_x_2.avg': {
+                avg: {
+                  field: 'rt_responsetime_x_2',
+                },
+              },
+            },
+          },
+          transformPreview: {
+            column: 0,
+            values: [`aal`, 'aca', 'amx', 'asa', 'awe'],
+          },
+          row: {
+            status: TRANSFORM_STATE.STOPPED,
+            mode: 'batch',
+            progress: '100',
+          },
+          indexPreview: {
+            columns: 10,
+            rows: 5,
+          },
+          indexDataPreview: [
+            { column: 4, name: 'rt_airline_lower' },
+            { column: 5, name: 'rt_responsetime_x_2' },
+          ],
+          previewData: [
+            { column: 0, name: 'rt_airline_lower', values: [`aal`, 'aca', 'amx', 'asa', 'awe'] },
+            { column: 1, name: 'rt_responsetime_x_2.avg' },
+            { column: 2, name: 'rt_responsetime_x_2.max' },
+            { column: 3, name: 'rt_responsetime_x_2.min' },
+          ],
+          discoverRuntimeFields: [
+            'rt_responsetime_x_2.avg',
+            'rt_responsetime_x_2.max',
+            'rt_responsetime_x_2.min',
+          ],
+        },
+      } as PivotTransformTestData,
+      {
+        type: 'latest',
+        suiteTitle:
+          'batch transform with unique rt_airline_lower and sort by time and runtime mappings',
+        source: 'ft_farequote',
+        uniqueKeys: [
+          {
+            identifier: 'rt_airline_lower',
+            label: 'rt_airline_lower',
+          },
+        ],
+        sortField: {
+          identifier: '@timestamp',
+          label: '@timestamp',
+        },
+        transformId: `fq_2_${Date.now()}`,
+        transformDescription: 'batch transform with unique rt_airline_lower and sort by time',
+        get destinationIndex(): string {
+          return `user-latest-${this.transformId}`;
+        },
+        expected: {
+          transformPreview: {
+            column: 0,
+            values: ['February 11th 2016, 23:59:54'],
+          },
+          row: {
+            status: TRANSFORM_STATE.STOPPED,
+            mode: 'batch',
+            progress: '100',
+          },
+          indexPreview: {
+            columns: 10,
+            rows: 5,
+          },
+          sourceIndex: 'ft_farequote',
+          indexDataPreview: [
+            { column: 2, name: 'airline' },
+            { column: 4, name: 'rt_airline_lower' },
+            { column: 5, name: 'rt_responsetime_x_2' },
+          ],
+          // The runtime mappings currently don't show up for Latest preview
+          // so no need to check
+          previewData: [],
```

Comment:
I reckon it's important to find out why. If it's a limitation of the ES API, we need to show a warning.

Reply:
This is currently intentional with our `preview` API so we decided to not show warnings, but we can revisit this in 7.13.